### PR TITLE
insmod: add page

### PR DESCRIPTION
**Pull request by shubham-kshetre**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/insmod.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# insmod
+
+> Dynamically loads modules into the Linux Kernel.
+> More information: <https://manned.org/insmod>.
+
+- Insert a kernel module into the Linux kernel:
+
+`insmod <module_name>.ko`
```
Comment (kbdharun) [inline pages/linux/insmod.md:8]:
```suggestion
`insmod {{module_name}}.ko`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have a suggestion for this page.

File: pages/linux/insmod.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# insmod
+
+> Dynamically loads modules into the Linux Kernel.
```
Comment (kbdharun) [inline pages/linux/insmod.md:3]:
```suggestion
> Dynamically load modules into the Linux Kernel.
```
Wdyt about this, I think `load` would better fit the description.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages/linux/insmod.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# insmod
+
+> Dynamically load modules into the Linux Kernel.
+> More information: <https://manned.org/insmod>.
+
+- Insert a kernel module into the Linux kernel:
+
+`insmod {{module_name}}.ko`
```
Comment (blueskyson) [inline pages/linux/insmod.md:8]:
```suggestion
`insmod {{path/to/module.ko}}`
```

**Review by blueskyson — APPROVED** [review]

**Pull request merged**